app/policies/external_activity_policy.rb

```diff
@@ -30,7 +30,7 @@ def set_private_before_matedit?
   end
 
   def copy?
-    user.present?
+    not_anonymous?
```

Comment:
Also makes sense to change this one: https://github.com/concord-consortium/rigse/blob/master/spec/policies/user_policy_spec.rb#L5 So we are always using the correct pattern.